src/ethereum/EtherscanButton/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Icon } from '../..';
+
+type Props = {
+  className?: string;
+  network?: 'mainnet' | 'rinkeby';
+  type: 'address' | 'tx';
+  value: string;
+};
+
+const StyledButton = styled.button`
+  background: none;
+  color: inherit;
+  border: none;
+  padding: 0;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+`;
+
+const EtherscanButton = ({
+  className,
+  type,
+  value,
+  network = 'mainnet',
+}: Props): React.ReactElement => {
+  const getNetwork = () => {
+    const lowerCaseNetwork = network.toLowerCase();
+    return lowerCaseNetwork === 'mainnet' ? '' : `${lowerCaseNetwork}.`;
+  };
+
+  const goToEtherscan = () =>
+    window.open(
+      `https://${getNetwork()}etherscan.io/${type}/${value}`,
+      '_blank'
+    );
+
+  return (
+    <StyledButton className={className} onClick={goToEtherscan}>
```

Comment:
why are we using a button for a link?

Reply:
This was implemented the right way in safe-react... https://github.com/gnosis/safe-react/blob/development/src/components/EtherscanBtn/index.tsx

please pay more attention to semantics of html elements you're using